ios/Classes/ComAppceleratorUrlSessionModule.m

```diff
@@ -128,17 +128,34 @@ -(id)addBackgroundUploadTask:(id)args
     
     ComAppceleratorUrlSessionSessionProxy *session = nil;
     NSString *url = nil;
-    TiBlob *blob = nil;
+    NSString *fileUrlString = nil;
+    NSString *method = nil;
+    NSURL *fileURL = nil;
+    NSDictionary *headers = nil;
     
     ENSURE_ARG_FOR_KEY(session, args, @"session", ComAppceleratorUrlSessionSessionProxy);
     ENSURE_ARG_FOR_KEY(url, args, @"url", NSString);
-    ENSURE_ARG_FOR_KEY(blob, args, @"data", TiBlob);
+    ENSURE_ARG_FOR_KEY(fileUrlString, args, @"file", NSString);
```

Comment:
`file` and `data` are supposed to do the same thing. Because Ti.Network.HTTPClient uses "data", we should make the type `id` and handle the type (`Ti.Blob` or `NSString`) afterwards to support both. But I can do that afterwards.